tests/util/transactions.ts

```diff
@@ -387,3 +391,13 @@ export const sendAllStreamAndWaitLast = async (
   }
   await Promise.all(promises);
 };
+
+// get supply factor
+export async function getSupplyFactor(context: DevTestContext) {
```

Comment:
```suggestion
export function getSupplyFactor(context: DevTestContext) {
```
Ideally this doesn't need to be an `async` function. 

Also looking into it, I'm not sure if it also needs to be exported, since it's mostly used locally within the file (unless I'm missing something).

Maybe can even think of inlining it once within the function body?

```
const supplyFactor = context.polkadotApi.runtimeVersion.specName.startsWith("moonbeam") ? MOONBEAM_SUPPLY_FACTOR: 1;
```